dist/linux/README.md

```diff
@@ -9,16 +9,30 @@ sudo make install
 
 ## Systemd Service (optional)
 
-Copy and enable the service:
+You can configure the program to run as systemd service, user- or system-wide (only one at a time).
+
+### As user service:
 
 ```sh
-cp byedpi.service ~/.config/systemd/user/
-systemctl --user daemon-reload
-systemctl --user enable byedpi.service
-systemctl --user start byedpi.service
+cp byedpi_user.service ~/.config/systemd/user/byedpi.service
+cp byedpi_options ~/.config/byedpi
+systemctl --user enable --now byedpi.service
 ```
 
 You should see the service now marked as "active":
 ```sh
 systemctl --user status byedpi.service
 ```
+
+### As system service:
+
+```sh
+sudo cp byedpi_system.service /usr/lib/systemd/system/byedpi.service
```

Comment:
`/usr/lib/` path should be used only by package maintainers.
For manual user global system install path is `/etc/`.

Reply:
I thoughtlessly copied it from my PKGBUILD repo. I'll correct it now.